packages/vue-renderer/src/renderer/ssr/index.js

```diff
@@ -0,0 +1,140 @@
+import path from 'path'
+import crypto from 'crypto'
+import fs from 'fs-extra'
+import devalue from '@nuxt/devalue'
+import { createBundleRenderer } from 'vue-server-renderer'
+
+import Renderer from '../index'
+
+export default class SSRRenderer extends Renderer {
+  get rendererOptions() {
+    const hasModules = fs.existsSync(path.resolve(this.context.options.rootDir, 'node_modules'))
+
+    return {
+      clientManifest: this.context.resources.clientManifest,
+      // for globally installed nuxt command, search dependencies in global dir
+      basedir: hasModules ? this.context.options.rootDir : __dirname,
+      ...this.context.options.render.bundleRenderer
+    }
+  }
+
+  renderScripts(context) {
+    return context.renderScripts()
```

Comment:
Why not directly calling `context` when needed?

Reply:
For common `render` function usage between `modernRenderer` and `ssrRenderer`.

If we call `context.renderScripts` in `render`, then `modernRenderer` has to override `render` which is a huge function

Reply:
Got it.